cmd/internal/cli/singularity.go

```diff
@@ -194,7 +194,7 @@ func setSylogColor() {
 func createConfDir(d string) {
 	if err := fs.Mkdir(d, os.ModePerm); err != nil {
 		if os.IsExist(err) {
-			sylog.Debugf("%s already exits. Not creating.", d)
+			sylog.Debugf("%s already exists. Not creating.", d)
```

Comment:
Optional...

```suggestion
			sylog.Debugf("%s already exists. Not creating", d)
```

It seems that most of the other debug messages dont have `.` at the end.

Reply:
Going to go ahead and merge since I think the period is appropriate given that this is a two "sentence" debug message